util/Migrator/DbScripts/2021-04-07_00_IncreaseOrgSeatSize.sql

```diff
@@ -0,0 +1,226 @@
+ALTER TABLE [dbo].[Organization]
+ALTER COLUMN [Seats] INT; 
```

Comment:
Is the `NULL` keyword not necessary in the `ALTER COLUMN` statement? (I'm not sure, obviously it worked in your local testing so I assume it's not but figured I would ask)
```sql
ALTER COLUMN [Seats] INT NULL;
```